politeiawww/api/cms/v1/v1.go

```diff
@@ -123,10 +127,15 @@ type InvoiceDetailsReply struct {
 // InvoiceInput is the expected structure of the invoice.json file being added to InvoiceRecords.
 // Users' raw csv will be inputted and parsed to help in their creation.
 type InvoiceInput struct {
-	ID        string           `json:"id"`    // Optional field for contractor ID entry
-	Month     uint16           `json:"month"` // Month of Invoice
-	Year      uint16           `json:"year"`  // Year of Invoice
-	LineItems []LineItemsInput `json:"lineitems"`
+	ID                 string           `json:"id"`                 // Optional field for contractor ID entry
+	Month              uint16           `json:"month"`              // Month of Invoice
+	Year               uint16           `json:"year"`               // Year of Invoice
+	ContractorName     string           `json:"contractorname"`     // IRL name of contractor
+	ContractorLocation string           `json:"contractorlocation"` // IRL location of contractor
+	ContractorContact  string           `json:"contractoremail"`    // Contractor email or other contact
+	ContractorRate     float64          `json:"contractorrate"`     // Contractor Pay Rate
```

Comment:
I'd prefer not to use JSON floats and deal with floats when doing money calculations. Any opposition to denominate rate in USD cents?